From fb44dd6ce0d3ad426dbfd7b63a62878d4dd69efb Mon Sep 17 00:00:00 2001
From: Christian Heimes <cheimes@redhat.com>
Date: Thu, 25 Feb 2016 10:42:51 +0100
Subject: [PATCH 1/3] Fix compile issue in RA_Token.cpp
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

GCC 6 is more strict than GCC 5 and earlier. It refuses to initialize a
char[] with 'int' initializers because char is signed while the its are
values from 0 to 255.

RA_Token.cpp:989:7: error: narrowing conversion of ‘194’ from ‘int’ to
‘char’ inside { }

https://fedorahosted.org/pki/attachment/ticket/2221
---
 base/tps-client/tools/raclient/RA_Token.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/tps-client/tools/raclient/RA_Token.cpp b/base/tps-client/tools/raclient/RA_Token.cpp
index dd5170c4c52b1234f7232ff585dfd6ca401e23cc..ec8307366bf226a27dc3eb739f5c1e78992e9a34 100644
--- a/base/tps-client/tools/raclient/RA_Token.cpp
+++ b/base/tps-client/tools/raclient/RA_Token.cpp
@@ -977,7 +977,7 @@ Sign (SECOidTag sigAlg, SECKEYPrivateKey * privKey, Buffer & blob)
         }
 
       /*fake proof for ECC until it works*/
-      char fake_proof [] = {
+      unsigned char fake_proof [] = {
             0x30 ,0x44 ,0x02 ,0x20 ,0x00,
             0xd6 ,0xc2 ,0x08 ,0x34 ,0x79 ,0x28 ,0x2e ,0x5f ,0x70 ,0xe5,
             0x38 ,0x1d ,0x84 ,0xa9 ,0x40 ,0x05 ,0x65 ,0x67 ,0x0f ,0x65,
-- 
2.5.0

